common/thorhelper/roxiehelper.ipp

```diff
@@ -87,6 +70,37 @@ interface IRoxieContextLogger : extends IContextLogger
     virtual bool collectingDetailedStatistics() const = 0;
 };
 
+class CRoxieContextLoggerHelper
```

Comment:
something not quite right with the existing #include structure.  ipp files should not be public, but this is included by roxiehelper.hpp.  Outside the scope of this PR to sort it out though - I have created HPCC-30284 to address it separately.